package.json

```diff
@@ -20,6 +30,7 @@
     "sax": "^1.4.1"
   },
   "devDependencies": {
-    "tape": "^4.6.3"
+    "rollup": "^4.21.1",
+    "tape": "^5.8.1"
```

Comment:
I upgraded tape as part of this PR in order to get support for running ESM test modules. This has the side effect of making the test suite a lot slower (~1m instead of ~12s). But apparently this is because they fixed a bug in their `deepEquals` assertion helper which we're using to assert that the fairly massive JSONs produced from each of the test cases are correct. So the tests are now slower but also stricter, which I suppose is a good thing.